azurerm/internal/services/apimanagement/resource_arm_api_management.go

```diff
@@ -201,6 +202,23 @@ func resourceArmApiManagementService() *schema.Resource {
 				},
 			},
 
+			"protocols": {
+				Type:     schema.TypeList,
+				Optional: true,
+				Computed: true,
```

Comment:
can we add a TODO to remove this in 2.0?

```suggestion
				Computed: true, //  TODO: remove in 2.0
```

Reply:
Added comment as suggested.